QuizApp/Presentation/HomeViewController.swift

```diff
@@ -60,6 +65,14 @@ extension HomeViewController: BindViewsProtocol {
             .$errorDescription
             .assign(to: \.errorDescription, on: errorPlaceholder)
             .store(in: &cancellables)
+
+        viewModel
+            .$filters
+            .receive(on: RunLoop.main)
```

Comment:
Nope, VM should already emit values on Main thread. So, ensure in VM that a new value is emitted on main.